gap_linux.go

```diff
@@ -53,6 +53,10 @@ func (a *Advertisement) Configure(options AdvertisementOptions) error {
 	for _, uuid := range options.ServiceUUIDs {
 		serviceUUIDs = append(serviceUUIDs, uuid.String())
 	}
+	var serviceData = make(map[string]interface{})
+	for uuid, data := range options.ServiceData {
+		serviceData[New16BitUUID(uuid).String()] = data.([]byte)
```

Comment:
This type assert doesn't seem to be necessary.
```suggestion
		serviceData[New16BitUUID(uuid).String()] = data
```